openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -262,7 +262,7 @@ def _create_and_set_jwt_cookies(response, request, user=None, refresh_token=None
         )
     else:
         access_token = create_dot_access_token(
-            request, user, oauth_application, expires_in=expires_in,
+            request, user, oauth_application, expires_in=expires_in, scopes=['email', 'profile'],
```

Comment:
Here is an example of the magic strings used in production code (i.e., not tests, and not a docstring).
  
Sorry, I just noticed them in several places and picked a random example that I saw to document the note, and it happened to be a docstring.  Oops. :)

Reply:
What's a magic string?

Reply:
@douglashall: "Magic numbers" and "magic strings" are terms for numbers and strings dropped in code that should be constants.  
https://deviq.com/magic-strings/